ios/FluentUI/Controls/Button.swift

```diff
@@ -120,26 +141,35 @@ open class Button: UIButton {
     open override var contentEdgeInsets: UIEdgeInsets {
         didSet {
             isUsingCustomContentEdgeInsets = true
+
+            if !isAdjustingCustomContentEdgeInsetsForImage && image(for: .normal) != nil {
```

Comment:
should we not do image inset when user is directly setting content insets of the image and title ? isUsingCustomContentEdgeInsets

Reply:
Custom content edge insets are still supported with this logic. We also don't have a choice to use that. It's impossible to add padding between the image and label without modifying the content edge insets (image edge insets and label edge insets alone cannot solve this).